R/rhtmlCombinedScatter.R

```diff
@@ -200,6 +200,12 @@
 #' @param label.placement.seed Label placement algorithm setting for the randomiser seed (Default is 1).
 #' @param label.placement.temperature.initial Label placement algorithm initial temperature (Default is 0.01).
 #' @param label.placement.temperature.final Label placement algorithm final temperature (Default is 0.0001).
+#' @param show.quadrants Whether to show quadrants
```

Comment:
Maybe reverse the order so that it is consistent with other params, e.g. `legend.show`, `labels.show`

Reply:
Thanks, done in https://github.com/Displayr/rhtmlCombinedScatter/pull/75/commits/3d3813abf51dbbee4b9e2429425b75be7953068f